src/main/java/org/fcrepo/spec/testsuite/crud/HttpPatch.java

```diff
@@ -122,12 +128,49 @@ public void serverManagedPropertiesModification(final String uri) {
                                         + "4.2.4.1), the server must fail the request by responding with a 4xx range"
                                         + " status code (e.g. 409 Conflict).",
                                         "https://fcrepo.github.io/fcrepo-specification/#http-patch", ps);
+
+        //This test assumes that creating an empty container will result in RDF that contains at least one
+        // server-managed triple.
         final Response resource = createBasicContainer(uri, info);
-        final String locationHeader = getLocation(resource);
+        final String resourceUri = getLocation(resource);
+
+        //parse the resource's body
+        final Model model = createDefaultModel();
+        model.read(doGet(resourceUri, new Header("Accept", "text/turtle")).getBody().asInputStream(), "",
+                   "text/turtle");
+        final Dataset dataset = new DatasetImpl(model);
```

Comment:
In this case I don't think you need to go through a Dataset, you should be able to operate on the model:
```
        final StmtIterator it = model.listStatements();
        boolean atLeastOneUpdateFailed = false;

        while (it.hasNext()) {
            //for each triple
            final Statement oldValue = it.next();
```

Reply:
Fair enough.